rpc/documentation/api.md

```diff
@@ -313,14 +317,18 @@ action.  Account discovery is typically only required when reseeding a wallet.
 - `bytes private_passphrase`: The private passphrase to unlock the wallet when
   account discovery is enabled.
 
+- `bytes starting_block_hash`: The hash at which Discover Addresses should 
+  begin.
```

Comment:
Reads better as "The hash of the first block checked for address and account usage."

Please also document that this must be a main chain block.